ui/src/components/EmailForm.vue

```diff
@@ -174,9 +315,33 @@ export default {
         cc: [],
         bcc: [],
         managerName: "", // TODO: use this field in the composed email later
-        managerEmail: "sender@xx.com" // TODO: ask the user to choose the sender email
-      }
+        managerEmail: "manager@xx.com", // TODO: ask the user to choose the sender email
+        reply_to:""
+      },
+      expand:false,
+      entityTypePanel: {
```

Comment:
Making these objects suggests that there is more state being managed per panel. If so, fine, but if not, consider making these attributes simple Boolean values. For example `showEntityTypePanel: false`, etc. (I'm guessing you've followed a pattern seen elsewhere in the code.)

Reply:
Ok, I see. I will change that coding pattern in the future.